src/c-unchop.c

```diff
@@ -1,211 +1,15 @@
-#include "rlang-types.h"
 #include "vctrs.h"
 
-enum fallback_homogeneous {
-  FALLBACK_HOMOGENEOUS_false = 0,
-  FALLBACK_HOMOGENEOUS_true
-};
-
 #include "decl/c-unchop-decl.h"
 
-
-static
-r_obj* list_unchop(r_obj* xs,
```

Comment:
Both `list_unchop()` and `vec_c()` at the C level are now thin wrappers around `list_combine()`